src/toast/ops/statistics.py

```diff
@@ -88,6 +90,13 @@ def _exec(self, data, detectors=None, **kwargs):
         log = Logger.get()
         nstat = 3  # Variance, Skewness, Kurtosis
 
+        if self.output_dir is not None:
```

Comment:
Why use `comm_world` here? Is it guaranteed that all process groups execute the operator at the same time?

Reply:
Fixed